src/cozy/react/components/ChangePasswordLink.jsx

```diff
@@ -32,8 +32,12 @@ const ChangePasswordLink = ({
     rawSettingsAppHref
   ).href
 
+  const app = {
+    slug: SETTINGS_APP_SLUG
+  }
+
```

Comment:
I think we don't care, but you create at every render a now app object, so you re-render AppLinker at each render of ChangePasswordLink rerender.

Reply:
You're right no reason not to do it